packs/data/feat-effects.db/stance-dread-marshal-stance-critical.json

```diff
@@ -0,0 +1,58 @@
+{
+    "_id": "i4aN13DzYCmrViVh",
+    "img": "systems/pf2e/icons/features/feats/dread-marshal-stance.webp",
+    "name": "Stance: Dread Marshal Stance (Critical)",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Dread Marshal Stance]{Dread Marshal Stance}.</p>\n<p>Your <a class=\"content-link\" draggable=\"true\" data-uuid=\"Compendium.pf2e.feat-effects.KBEJVRrie2JTHWIK\" data-id=\"KBEJVRrie2JTHWIK\" data-type=\"Item\" data-pack=\"pf2e.feat-effects\" data-tooltip=\"Effect Item\">Marshal's Aura</a> increases to a <span class=\"with-repost\" data-pf2-effect-area=\"emanation\" data-pf2-distance=\"20\" data-pf2-traits=\"archetype,open,stance\">20-foot emanation</span>, and it grants you and allies a status bonus to damage rolls equal to the number of weapon damage dice of the unarmed attack or weapon you are wielding that has the most weapon damage dice. When you or an ally in the aura critically hits an enemy with a Strike, that enemy is <a class=\"content-link\" draggable=\"true\" data-uuid=\"Compendium.pf2e.conditionitems.TBSHQspnbcqxsmjL\" data-id=\"TBSHQspnbcqxsmjL\" data-type=\"Item\" data-pack=\"pf2e.conditionitems\" data-tooltip=\"Condition Item\" aria-describedby=\"tooltip\">Frightened 1</a>.</p>"
```

Comment:
Looks like some cruft ended up in this description.

Reply:
Cleaned up